### PR TITLE
✨ added support for consent for Taboola amp-analytics

### PR DESCRIPTION
**Pull request by udaya-m-taboola**
Our publisher partners want us to honor the user consent flag. This PR is to add support in our amp-analytics implementation.

### CODE REVIEW

File: extensions/amp-analytics/0.1/vendors/taboola.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "aid": ""
   },
   "requests": {
-    "loadamptrkhtml": "https://c3.taboola.com/amp/amp-analytics-taboola.html?url=${canonicalUrl}&aid=${aid}"
+    "loadamptrkhtml": "https://c3.taboola.com/amp/amp-analytics-taboola.html?url=${canonicalUrl}&aid=${aid}&consent=${consent}"
```
Comment (micajuine-ho) [inline extensions/amp-analytics/0.1/vendors/taboola.json:6]: Where is this `${consent}` variable defined? Or are you trying to use AMP predefined consent variable substitutions?

Comment (udaya-m-taboola) [reply extensions/amp-analytics/0.1/vendors/taboola.json:6]: @micajuine-ho yes using the predefined consent variable substitutions. The idea is our publisher partners will setup the AMP predefined consent variable.

Comment (micajuine-ho) [reply extensions/amp-analytics/0.1/vendors/taboola.json:6]: I believe you are looking for theses variables: https://github.com/ampproject/amphtml/blob/main/extensions/amp-analytics/0.1/variables.js#L321-L327

Did you see `${consent}` anywhere in our documentation?

Comment (udaya-m-taboola) [reply extensions/amp-analytics/0.1/vendors/taboola.json]: @micajuine-ho fixed the substitution names. sorry about that.

**Review by micajuine-ho — APPROVED** [review]

**Comment by udaya-m-taboola** [comment]
@micajuine-ho rebased it. Should be ready to merge

**Comment by udaya-m-taboola** [comment]
@micajuine-ho does the rebase look good for the merge ?

**Pull request merged**